### PR TITLE
Add the role in the print requests

### LINKED ISSUE #1182: Missing role attribute
The theme and the print capabilities requests should have a parameter with the role id to have a cache per role.

### PR DESCRIPTION
**Pull request by sbrunner**
fix #1182

### CODE REVIEW

**Comment by sbrunner** [comment]
Demo: https://camptocamp.github.io/ngeo/print-role/examples/contribs/gmf/apps/desktop_alt

**Comment by ger-benjamin** [comment]
LGTM

File: contribs/gmf/src/services/authenticationservice.js [hunk 1]
```diff
@@ -231,6 +231,14 @@ gmf.Authentication.prototype.resetPassword = function(login) {
 
 
 /**
+ * @return {number|null} The role ID.
```
Comment (ger-benjamin) [inline contribs/gmf/src/services/authenticationservice.js]: Please add `@export` (with a little description)

**Comment by ger-benjamin** [comment]
One little fix then LGTM

File: contribs/gmf/src/services/authenticationservice.js (hunk 1 above)
Comment (sbrunner) [inline contribs/gmf/src/services/authenticationservice.js]: Finally I remove a other `@export` :-)

**Pull request merged**